apiserver/bundle/bundle_test.go

```diff
@@ -1,26 +1,47 @@
-// Copyright 2015 Canonical Ltd.
+// Copyright 2016 Canonical Ltd.
 // Licensed under the AGPLv3, see LICENCE file for details.
 
-package client_test
+package bundle_test
```

Comment:
:x: please don't use export_test... if the only thing that it is needed is to access an newFacade, just put this test in package bundle, and use the unexported function directly.